examples/terraform/modules/bootkube/bootkube.tf

```diff
@@ -11,5 +11,6 @@ module "bootkube" {
   kube_apiserver_service_ip     = "${var.k8s_apiserver_service_ip}"
   kube_dns_service_ip           = "${var.k8s_dns_service_ip}"
   kube_etcd_service_ip          = "${var.k8s_etcd_service_ip}"
+  container_linux_oem           = "${var.container_linux_oem}"
```

Comment:
The bootkube-terraform module just renders asset files, it doesn't take this value.